### ISSUE TITLE
Failed to push to github

### ISSUE THREAD

**Issue by lmame** [stackblitz/core#958]
I am at ngConf 2019 and was working on a workshop.
I took the trainer stackblitz, forked it and used stackblitz to create a repo on my github.
It went ok.
Locally on my laptop I cloned the repo and added the slides (20MB+?).
On stackblitz I pulled the modification (no error message).
I worked on the workshop and tried to commit.
It failed with an error message (not simple enough for fast forward, use force).

Then now the stackblitz project cannot even load anymore:
https://stackblitz.com/edit/ngconf2019-bz-angular-ngrx

Import error, can't find files:
/src/main.ts
/src/index.html
/src/styles.css
/src/polyfills.ts
/src/app/app.module.ts
/src/app/app.routing.ts
/src/app/app.component.ts
/src/app/app.component.css
/src/app/app.component.html
/src/app/hello.component.ts
/src/app/banana/banana.component.ts
/src/app/banana/banana.component.css
/src/app/banana/banana.component.html

**Comment by AmmonK** [comment]
I received the same error with a JavaScript project.  I hard-reloaded the page and was able to make my commits after that.  I'm not sure if Git has a different state for when the project is first connected vs when it is reloaded.  This might not work for you since it's a different project type but at least in this scenario it worked.

**Comment by joselvelez** [comment]
I am getting the same issue on a tutorial that I am playing around with. Using Angular.

**Comment by dimitrileite** [comment]
What is the workaround for that, please?

**Comment by willi84** [comment]
@lmame I see this when I click your link:

![image](https://user-images.githubusercontent.com/6207308/58870930-656d9000-86c1-11e9-9499-d0b82a114c06.png)

**Comment by krishnakakade1999** [comment]
You can download that folder on your local computer and then push to the github ok that is also way or you can fork bt i am not able to fork that ok i hope this help

**Comment by menaka-siri** [comment]
It seems that StackBlitz fails to push the commits to Github when you try to click on commit button after few hours, I think. I couldn't track the exact time period.
But when I push commits with short time intervals (within minutes), the code changes have been successfully pushed to the repo.

**Comment by seiyria** [comment]
One solution that worked for me was to make a new branch and merge them via PR on github. I would like to just be able to force push always though, as I have no intention of editing this project outside of stackblitz.

**Comment by massimocoletti** [comment]
is it possible to perform commit with additional parameters?

**Comment by ashishkumar-xx** [comment]
Please update when this issue got fixed from stackblitz

**Comment by jeserkin** [comment]
Okay. Found a dirty workaround.
1. Chose different branch from branch list (e.g. New branch)
2. Stash changes into new branch (it will only close base branch and switch to it)
3. Now commit existing changes into new branch
4. On Github create PR on new branch
5. Merge PR into base branch
6. Delete tmp branch
7. In stackblitz switch to base branch and pull changes
8. Profit!!!??? Not really.

**Comment by yuwash** [comment]
Yes, @jeserkin’s method (stash) is the only thing that worked for me so far. Neither pulling nor creating a new branch helped (in a Javascript project).

**Comment by jeserkin** [comment]
I still feel like it could be solved more elegantly.

**Comment by giniblanco** [comment]
I just started with Github and Angular, but I had to fork it to make it do something and then it allowed me to connect to my repository and commit it.

**Comment by lmame** [comment]
Thanks @kc0tlh :)
